centos_ci.sh

```diff
@@ -27,19 +27,8 @@ yum -y install \
   curl \
   python-setuptools
 
-
-# Install Docker (https://docs.docker.com/engine/installation/linux/centos/#install-with-yum)
-sudo tee /etc/yum.repos.d/docker.repo <<-'EOF'
-[dockerrepo]
-name=Docker Repository
-baseurl=https://yum.dockerproject.org/repo/main/centos/7/
-enabled=1
-gpgcheck=1
-gpgkey=https://yum.dockerproject.org/gpg
-EOF
-
-yum -y install docker-engine
-systemctl enable docker.service
+# Install docker
+yum install -y docker
```

Comment:
@budhrg I think it is good idea to keep the `systemctl enable docker.service`. In case you want the docker to be running after a reboot.